### PR TITLE
MVP

### PR DESCRIPTION
**Pull request by BohdanDudkaTD**
This PR adds an implementation of the MVP by adding `presenters` who create corresponding pages and sends requests to the server using `RequestService`. This class implements HTTP methods which might be sent to the server (GET, POST, etc.).

Result of the ESLint execution:
image

### CODE REVIEW

**Comment by BohdanDudkaTD** [comment]
@dkalinin, PTAL

File: app/view/component/registrationform/index.js [hunk 1]
```diff
@@ -27,6 +27,15 @@ export class RegistrationForm {
     this._callback = callback;
   }
 
+  /**
+   * Get an instance of the {@link Button} component created on this form.
+   *
+   * @return {Button} buttons instance.
+   */
+  get button() {
```
Comment (dkalinin) [inline app/view/component/registrationform/index.js]: Button should be private.

File: app/view/component/loginform/index.js [hunk 2]
```diff
@@ -25,6 +26,15 @@ export class LogInForm {
     this._callback = callback;
   }
 
+  /**
+   * Get an instance of the {@link Button} component created on this form.
+   *
+   * @return {Button} buttons instance.
+   */
+  get button() {
```
Comment (dkalinin) [inline app/view/component/loginform/index.js]: Button should be private.

File: app/view/page/loginpage/index.js [hunk 3]
```diff
@@ -35,5 +75,11 @@ export class LogInPage {
     loginHref.title = 'Register';
     loginHref.href = '#/registration';
     this._root.appendChild(loginHref);
+
+    const errorSpan = document.createElement('span');
+    errorSpan.textContent = 'Sorry, try to login again';
```
Comment (dkalinin) [inline app/view/page/loginpage/index.js]: ... to log in again...

File: app/view/page/registrationpage/index.js [hunk 4]
```diff
@@ -15,6 +15,40 @@ export class RegistrationPage {
...
+   *
+   * @param {function} callback - the callback function which will be called when action occurs.
+   * @public
+   */
+  onUserCreated(callback) {
+    this._callback = callback;
+  }
+
+  /**
+   * Lock (disable) submit button on the form so the user can`t click it.
```
Comment (dkalinin) [inline app/view/page/registrationpage/index.js]: Please follow convention for methods documentations: Disable-s- submit button...

Comment (dkalinin) [reply app/view/page/registrationpage/index.js]: Please fix in all JSDoc for methods.

**Comment by BohdanDudkaTD** [comment]
@dkalinin, PTAL, fixed issues:
- encapsulate logic to lock/unlock form when the user sends a request to the server;
- grammar issues in JS docs;

**Pull request merged**